To get the test to run I had to change the end_to_end model to `together/openai/gpt-oss-20b`

When I do this I don't seem to get the expected completion:

```python
 @skip_if_no_together
    async def test_together_stream_end_to_end() -> None:
        model = get_model(
            "together/openai/gpt-oss-20b",
            stream=True,
            config=GenerateConfig(max_tokens=50, temperature=0.0),
        )
        response = await model.generate(
            input=[ChatMessageUser(content="This is a test string. What are you?")]
        )
>       assert len(response.completion) >= 1
E       AssertionError: assert 0 >= 1
E        +  where 0 = len('')
E        +    where '' = ModelOutput(model='openai/gpt-oss-20b', choices=[ChatCompletionChoice(message=ChatMessageAssistant(id='QrBBSYD798btkTc...put_tokens_cache_read=None, reasoning_tokens=None, total_cost=None), time=1.135119707789272, metadata=None, error=None).completion

tests/model/providers/test_openai_compatible.py:399: AssertionError
```

--- a/tests/model/providers/test_openai_compatible.py
+++ b/tests/model/providers/test_openai_compatible.py
@@ -321,10 +321,13 @@ def test_together_stream_model_arg_forwarded(stream: bool) -> None:
 
 @skip_if_no_together
 async def test_together_stream_end_to_end() -> None:
+    # Use a generous max_tokens: reasoning models (e.g. gpt-oss) spend their
+    # budget on the reasoning channel and emit no answer content if it is too
+    # low, producing an empty completion (regardless of streaming).
     model = get_model(
-        "together/meta-llama/Llama-3.1-8B-Instruct-Turbo",
+        "together/openai/gpt-oss-20b",
         stream=True,
-        config=GenerateConfig(max_tokens=50, temperature=0.0),
+        config=GenerateConfig(max_tokens=1024, temperature=0.0),
     )
     response = await model.generate(
         input=[ChatMessageUser(content="This is a test string. What are you?")]

